packages/fast-layouts-react/src/utilities/breakpoint-tracker.ts

```diff
@@ -91,6 +91,13 @@ class BreakpointTracker {
         this.openRequestAnimationFrame = false;
     };
 
+    /**
+     * Returns the current breakpoint
+     */
+    public getCurrentBreakpoint = (): Breakpoint => {
```

Comment:
naming nit - not sure you need the 'get' part of the method - I think `currentBreakpoint` makes it pretty clear it is returning the current breakpoint

Reply:
👍